mergeAOSP.sh

```diff
@@ -1,34 +1,18 @@
+## This script simplifies merge of AOSP branches to the JetBrains fork.
+## The JetBrains fork contains multiple files/folders that are fully developed independently of AOSP,
+## and should not be updated by the upstream version
+
 #!/bin/bash
```

Comment:
As far as I remember, it's important in some cases to have this line first. Please move it to the beginning of the file.

Reply:
Done